README.md

```diff
@@ -102,29 +98,33 @@ The `setup` command will install the downloaded tarballs to the directory set by
 env.sh and run you local development cluster. The command can be run in several different ways:
 
 1. Sets up Apache Accumulo and its dependencies of Hadoop, Zookeeper. This starts all processes and
-   will wipe Accumulo/Hadoop if this command was run previously. This command also sets up Spark
-   and starts Spark's History Server (set `START_SPARK_HIST_SERVER=false` in your env.sh to turn 
-   off). This command is useful if you are using Uno for Accumulo development.
+   will wipe Accumulo/Hadoop if this command was run previously.
 
         uno setup accumulo
 
-2. Sets up Apache Fluo along with Accumulo (and its dependencies). It also sets up a metrics server
-   for Fluo consisting of InfluxDB & Grafana if `SETUP_METRICS` is set to true in env.sh. This
-   command will wipe your cluster. While Fluo is set up, it does not start any Fluo applications.
+2. Sets up Apache Fluo along with Accumulo (and its dependencies). This command will wipe your
+   cluster. While Fluo is set up, it does not start any Fluo applications.
 
-        uno setup fluo
+        uno setup accumulo fluo
 
 3. Sets up Apache Fluo only. This will stop any previously running Fluo applications but it will not
    wipe your cluster. If you want upgrade Fluo without wiping your cluster, run `uno fetch fluo`
    before running this command.
 
-        uno setup fluo-only
+        uno setup fluo
```

Comment:
That sounds good to me.  However, I would like to shorten it to `--no-deps` and I think I will only do it for Fluo at this time.